TSPL.docc/LanguageGuide/Concurrency.md

```diff
@@ -564,33 +637,192 @@ the explicit parent-child relationships between tasks
 let Swift handle some behaviors like propagating cancellation for you,
 and lets Swift detect some errors at compile time.
 
+[`TaskGroup`]: https://developer.apple.com/documentation/swift/taskgroup
+
+<!-- TR: What's a good example of this kind of error? -->
+
+Here's another version of the code to download photos
+that handles any number of photos:
+
 ```swift
 await withTaskGroup(of: Data.self) { taskGroup in
```

Comment:
the Data here means that the child task returns Data -- but the `addTask{}` does not seem to do so - this would likely not compile.

```suggestion
await withTaskGroup(of: Data.self) { group in
```

i'd suggest calling it just group?